kubernetes-examples/src/main/java/io/fabric8/kubernetes/examples/PortForwardExample.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.fabric8.kubernetes.examples;
+
+import io.fabric8.kubernetes.api.model.Pod;
+import io.fabric8.kubernetes.client.Config;
+import io.fabric8.kubernetes.client.ConfigBuilder;
+import io.fabric8.kubernetes.client.DefaultKubernetesClient;
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.kubernetes.client.LocalPortForward;
+
+import okio.ByteString;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.InetSocketAddress;
+import java.nio.ByteBuffer;
+import java.nio.channels.SocketChannel;
+
+public class PortForwardExample {
+  private static final Logger logger = LoggerFactory.getLogger(PortForwardExample.class);
+
+  public static void main(String args[]) {
+    String master = "https://localhost:8443";
+
+    Config config = new ConfigBuilder().withMasterUrl(master).build();
+    try (final KubernetesClient client = new DefaultKubernetesClient(config)) {
+      String namespace = "default";
+      log("namespace", namespace);
+      Pod pod = client.pods().inNamespace(namespace).load(PortForwardExample.class.getResourceAsStream("/portforward-example-pod.yml")).get();
+      log("Pod created");
```

Comment:
shouldn't this be logged after the next line ? after the client has created the resource in the desired namespace ?

@rohanKanojia @kunal-kushwaha

Reply:
yeah, makes sense